src/vendor/processors/authorizenet.py

```diff
@@ -849,6 +883,60 @@ def get_customer_id_for_expiring_cards(self, month):
 
         return customer_profile_ids
     
+    def get_customer_and_payemnt_id_for_expiring_cards(self, month):
```

Comment:
typo: *get_customer_and_payment_id_for_expiring_cards

here and in test_authorizenet

Reply:
thank you !!!!